smac/smbo/smbo.py

```diff
@@ -102,7 +103,11 @@ def run(self):
             The best found configuration
         '''
         self.stats.start_timing()
-        self.incumbent = self.initial_design.run()
+        try:
+            self.incumbent = self.initial_design.run()
+        except TAEAbortException:
```

Comment:
This exception changes the current behaviour of SMAC as it does not indicate a failure if the first run aims to abort the evaluation. Thus, I think this try/except is not necessary at this point and should be only added in a PR to fix #160 (or this PR should also fix #160).